components/bl-audio-player-component/README.md

```diff
@@ -1,25 +1,51 @@
 # Audio Player
 
-This is a component for Backendless [UI-Builder](https://backendless.com/developers/#ui-builder) designer. You will have a styled player with the set of your tracks. It supports one or multiple audio sources and has all functions you can think of.
-
-While the player can be configured with multiple settings it is easy in use and provides users with smooth experience.
+The Audio Player component provides basic functionality to play audio files. 
 
 <p align="center">
   <img src="./thumbnail.png" alt="main thumbnail" width="780"/>
 </p>
 
+For every audio file that you want to run in the player, you must provide a valid URL where the audio file is located.
+
+There are two ways to specify the audio file URL in the Settings of this component:
+ 1. Enter a string value in the **Audio Url** property. The entered value must be the URL to the audio file.
+ 2. Create a list containing URL addresses in the Logic of this component.
+
+The valid format of the URL address looks as following: ````https://mycloudservice.com/audio/ambience/forest_sounds.mp3````
```

Comment:
It looks like real URL but it doesn't work. Could we change it for mock URL like `https://some-site.com/audio-name.mp3` or use the working one?

Reply:
Will be ````https://example-cloudservice.com/audio/ambience/forest_sounds.mp3````